doc/howtos/vulnerability-management.rst

```diff
@@ -0,0 +1,145 @@
+Ostro OS vulnerability handling
+###############################
+
+Introduction
+============
+
+Software vulnerabilities are found continuously even in mature pieces
+of software. This means that one of the key objectives of a Linux
+distribution is pushing fixes to vulnerabilities down to the end users.
+Ostro OS images will not be used as-is by end users, but Ostro OS
+will be the base software distribution for many products the Ostro OS
+customers are developing. These products must be kept secure for their
+expected lifetime. The published Ostro OS images are for development
+purposes only and should not be used in real products.
```

Comment:
Should probably have a sentence or two here describing how real products should be deployed with Ostro so readers know how it differs from the Ostro images without having to search for this information themselves.  Something like "The development images do not auto-update.  Deployed products should be built from Ostro sources and contain a method for updating as needed; see $link for more details on the expected path for deploying Ostro in products."